app/components/crate-report-form.module.css

```diff
@@ -0,0 +1,69 @@
+.report-form {
+  background-color: var(--main-bg);
+  padding: 0.5rem 1rem;
+}
+
+.form-group {
+  border: none;
+  margin: 0;
+  padding: 0;
+
+  & + & {
+    margin-top: 1rem;
+  }
+}
+
+.form-group-name {
+  composes: form-group-name from '../styles/settings/tokens/new.module.css';
+}
+
+.reasons-list {
+  composes: scopes-list from '../styles/settings/tokens/new.module.css';
+  label {
+    flex-wrap: nowrap;
+  }
+  input {
+    align-self: center;
+  }
+}
+
+.detail {
+  border: 1px solid var(--gray-border);
+  border-radius: var(--space-3xs);
+  resize: vertical;
+  width: 100%;
+
+  &.invalid {
+    background: light-dark(#fff2f2, #170808);
+    border-color: red;
+  }
+}
+
+.form-group-error {
+  composes: form-group-error from '../styles/settings/tokens/new.module.css';
+}
+
+.buttons {
+  composes: buttons from '../styles/settings/tokens/new.module.css';
+  justify-content: end;
+  gap: 2rem;
+}
+
+.button {
+  &:focus {
+    outline: 1px solid var(--bg-color-top-dark);
+    outline-offset: 2px;
+  }
```

Comment:
I also included an outline for focused buttons, as I previously had difficulty distinguishing them visually. (IMHO, we might also want to apply this to some other buttons, especially those with gradients.)

Reply:
yeah, the buttons never had great a11y...

we could try adding that to all of the buttons from the shared module, but that should probably be done in a dedicated PR :)

Reply:
That would be great! And it definitely should be done in separate PRs.